ui/selectors/selectors.js

```diff
@@ -3052,14 +3052,15 @@ export function getUsePhishDetect(state) {
  * Gets the cached address security alert response for a given address
  *
  * @param {*} state
- * @param {string} address - The address to get security alert for
- * @returns the cached address security alert response for the given address
+ * @param {string} cacheKey - The cache key in format "chainId:address" to get security alert for
```

Comment:
```suggestion
 * @param {string} cacheKey - The cache key in format "chain:address" to get security alert for
```

Since `createCacheKey` uses the chain name rather than the chain ID.